quark.h

```diff
@@ -273,9 +303,11 @@ struct quark_event {
 #define QUARK_EV_SETPROCTITLE		(1 << 3)
 #define QUARK_EV_SOCK_CONN_ESTABLISHED	(1 << 4)
 #define QUARK_EV_SOCK_CONN_CLOSED	(1 << 5)
+#define QUARK_EV_PACKET			(1 << 6)
 	u64				 events;
 	const struct quark_process	*process;
 	const struct quark_socket	*socket;
+	struct quark_packet		*packet;
```

Comment:
why not const?

Reply:
Cause otherwise I can't free it without casting, and I think the casting is worse since I'll lose the type.
I'd rather never lose the type.

```
quark.c: In function ‘event_storage_clear’:
quark.c:385:31: error: passing argument 1 of ‘free’ discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
  385 |         free(qq->event_storage.packet);
      |              ~~~~~~~~~~~~~~~~~^~~~~~~
In file included from quark.c:21:
/usr/include/stdlib.h:687:25: note: expected ‘void *’ but argument is of type ‘const struct quark_packet *’
  687 | extern void free (void *__ptr) __THROW;
      |                   ~~~~~~^~~~~
```